src/services/static-snapshot-options.ts

```diff
@@ -3,5 +3,5 @@ export interface StaticSnapshotOptions {
     port: number,
     baseUrl: string,
     snapshotFilesRegex: string,
-    ignoreFilesRegex?: string,
+    ignoreFilesRegex: string,
```

Comment:
This was always passed to the snapshot service since it has a default value, so it was never really optional.